index.js

```diff
@@ -0,0 +1,89 @@
+/* eslint-disable semi */
+const bookTitle = document.getElementById('book-title');
+const authorName = document.getElementById('book-author');
+const button = document.getElementById('submit');
+const list = document.querySelector('#books-ul');
+// const array = [];
+// let counter = 0;
+// let delcounter = 0;
+
+class Collection {
+  constructor() {
+    this.bookArray = [];
+    this.counter = 0;
+    this.delcounter = 0;
+  }
+
+  addBook(book, author) {
+    const object = {};
+    object.id = this.counter;
+    object.title = book;
+    object.author = author;
+    this.bookArray.push(object);
+    this.display(book, author);
+  }
+
+  display(book, author) {
+    const li = document.createElement('li');
+    const removebtn = document.createElement('button');
+    li.setAttribute('id', `${this.counter}`);
+    li.className = 'lists';
+    li.textContent = `${book} by ${author}`;
+    // REMOVE BUTTON
+    removebtn.className = 'removebuttons'
+    removebtn.textContent = 'Remove';
+    // removebtn.addEventListener('click', this.removeBook);
```

Comment:
- [ ] I would like to recommend you remove all the commented code from your codebase in order to keep your code clean :smile: